app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,47 @@
+class NotesController < ApplicationController
+
+  def index
+    find_initiative
+    redirect_to user_initiative_path(team_member, @initiative)
+  end
+
+  def new
+    find_initiative
+    @note = Note.new
+  end
+
+  def create
+    find_initiative
+    @note = @initiative.notes.build(note_params.merge(initiative_id: @initiative_id, user_id: current_user.id))
```

Comment:
Because you're building the notes off of an initiative, does the initiative_id need to be passed in as a param? If so, something might not be working correctly about that.

Reply:
Resolved!